# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/unreal-ircd/hotfix-noroot.patch
# Copyright (C) 2025 - 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- unrealircd-6.1.10/Config.vanilla	2025-08-04 22:48:04.738183480 +0200
+++ unrealircd-6.1.10/Config	2025-08-04 22:48:10.122406226 +0200
@@ -327,14 +327,6 @@
 	ADVANCED=""
 fi
 
-if [ "`id -u`" = "0" ]; then
-	echo "ERROR: You cannot build or run UnrealIRCd as root"
-	echo ""
-	echo "Please create a separate account for building and running UnrealIRCd."
-	echo "See https://www.unrealircd.org/docs/Do_not_run_as_root"
-	exit
-fi
-
 # Check for gmake early...
 if [ "$MAKE" = "" ]; then
 	MAKE="make"
